src/cheri/debug-integration.adoc

```diff
@@ -204,7 +204,20 @@ The <<m_bit>> of this capability is _only_ updated by executing <<MODESW_CAP>> o
 include::img/drootcreg.edn[]
 
 ==== Modified Trap-Return Instruction Behavior
-The `DRET` instruction reads the full YLEN bits of the <<mepc_y>>/<<sepc_y>> register and unseals it prior to exception return if it is a <<sentry_cap>>.
+
+#Begin changed since last ARC review#
+
+If the platform incorporates <<section_rvy_sentry_jalr_ext>>,
+the `DRET` instruction reads the full YLEN bits of the <<mepc_y>>/<<sepc_y>> register and unseals it prior to exception return,
```

Comment:
I think this can be independent of this new extension. Can we say: "DRET unseals all ambient sentry types"? Once we have Zycfi, we could have a table of capability types that are unsealed by JALR+xRET?

Reply:
In the case of `Xycheriot`, the types used by `JALR` are _not_ ambient: we explicitly do not want `YBLD` to build them.  More generally, even if you believe _forward_ arc sentry types should be ambient, _backward_ arc sentry types are probably not, on the grounds that one wants to ensure that they come about only from `JALR` itself (and TCB code via `YSEAL`, perhaps).

I'll try to make this less chatty, especially with `Zysentry`'s hooks moving into the base ISA.

Reply:
Thank you that sounds great!